client/python/kserve-api/samples/README.md

```diff
@@ -569,13 +558,146 @@ optional arguments:
                         Add demultiplexer dimension at front
 ```
 
-<!---
-TO DO: Complete and uncomment when endpoint ready
-
 - Usage Example
 
 ```Bash
-TODO
+python3 ./http_infer_resnet.py --http_port 5000 --images_numpy_path ../../imgs.npy --labels_numpy_path ../../lbs.npy --input_name 0 --output_name 1463 --model_name resnet --transpose_input False
+Image data range: 0.0 : 255.0
+Start processing:
+        Model name: resnet
+        Iterations: 10
+        Images numpy path: ../../imgs.npy
+        Numpy file shape: (10, 3, 224, 224)
+
+Iteration 1; Processing time: 206.35 ms; speed 4.85 fps
+imagenet top results in a single batch:
+         0 airliner 404 ; Correct match.
+Iteration 2; Processing time: 196.61 ms; speed 5.09 fps
+imagenet top results in a single batch:
+         0 Arctic fox, white fox, Alopex lagopus 279 ; Correct match.
+Iteration 3; Processing time: 184.16 ms; speed 5.43 fps
+imagenet top results in a single batch:
+         0 bee 309 ; Correct match.
+Iteration 4; Processing time: 189.03 ms; speed 5.29 fps
+imagenet top results in a single batch:
+         0 golden retriever 207 ; Correct match.
+Iteration 5; Processing time: 188.47 ms; speed 5.31 fps
+imagenet top results in a single batch:
+         0 gorilla, Gorilla gorilla 366 ; Correct match.
+Iteration 6; Processing time: 192.66 ms; speed 5.19 fps
+imagenet top results in a single batch:
+         0 magnetic compass 635 ; Correct match.
+Iteration 7; Processing time: 175.03 ms; speed 5.71 fps
+imagenet top results in a single batch:
+         0 peacock 84 ; Correct match.
+Iteration 8; Processing time: 192.43 ms; speed 5.20 fps
+imagenet top results in a single batch:
+         0 pelican 144 ; Correct match.
+Iteration 9; Processing time: 190.80 ms; speed 5.24 fps
+imagenet top results in a single batch:
+         0 snail 113 ; Correct match.
+Iteration 10; Processing time: 210.31 ms; speed 4.75 fps
+imagenet top results in a single batch:
+         0 zebra 340 ; Correct match.
+
+processing time for all iterations
+average time: 192.20 ms; average speed: 5.20 fps
+median time: 191.00 ms; median speed: 5.24 fps
+max time: 210.00 ms; min speed: 4.76 fps
+min time: 175.00 ms; max speed: 5.71 fps
+time percentile 90: 206.40 ms; speed percentile 90: 4.84 fps
+time percentile 50: 191.00 ms; speed percentile 50: 5.24 fps
+time standard deviation: 9.58
+time variance: 91.76
+Classification accuracy: 100.00
+```
+
+
+### Run the Client to perform inference with binary encoded image <a name="http-model-infer-binary"></a>
+
+Using binary inputs feautre requires model to accept input in layout NHWC. The model used in other clients has native layout NCWH, therefore it must be adjusted on model server start up. For samples that use binary inputs, please start docker container with `--layout NHWC:NCHW` parameter:
```

Comment:
this requirement is only for jpeg/png encoded input. With other types of data, the layout might be arbitrary.